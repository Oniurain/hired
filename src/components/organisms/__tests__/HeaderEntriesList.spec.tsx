import * as React from 'react'
import { shallow } from 'enzyme'

import { HeaderEntriesList } from '../'

describe('HeaderEntriesList', () => {
  it('should render', () => {
    const subject = shallow(
      <HeaderEntriesList
        project="ACME Inc."
      />
    )

    expect(subject).toMatchSnapshot()
  })
})
