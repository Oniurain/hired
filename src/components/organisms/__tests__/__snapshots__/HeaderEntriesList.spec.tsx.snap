// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderEntriesList should render 1`] = `
<styled.header>
  <Styled(WithRoute[BaseLink])
    routeName="PROJECT_LIST"
  >
    <Styled(Icon) />
  </Styled(WithRoute[BaseLink])>
  <styled.h1>
    ACME Inc.
  </styled.h1>
  <div>
    Â 
  </div>
</styled.header>
`;
